ql/src/java/org/apache/hadoop/hive/ql/exec/FileSinkOperator.java

```diff
@@ -616,6 +617,8 @@ protected void initializeOp(Configuration hconf) throws HiveException {
       initializeSpecPath();
       fs = specPath.getFileSystem(hconf);
 
+      hconf.set(WRITE_OPERATION_CONFIG_PREFIX + getConf().getTableInfo().getTableName(),
```

Comment:
Could we just do this like:
`HiveCustomStorageHandlerUtils.setWrite(hconf, tableName)`?

Reply:
Moved both get/set write operation to HiveCustomStorageHandlerUtils.